dist/angular-ui-notification.js

```diff
@@ -52,6 +53,7 @@ angular.module('ui-notification').provider('Notification', function() {
             args.positionY = args.positionY ? args.positionY : options.positionY;
             args.positionX = args.positionX ? args.positionX : options.positionX;
             args.replaceMessage = args.replaceMessage ? args.replaceMessage : options.replaceMessage;
+            args.closeOnClick = args.closeOnClick ? args.closeOnClick : options.closeOnClick;
```

Comment:
Wouldn't this be better:
args.closeOnClick = (args.closeOnClick !== null && args.closeOnClick !== undefined) ? args.closeOnClick : options.closeOnClick;

because args.closeOnClick = false will always make args.closeOnClick = options.closeOnClick.